app/forms/prior_authority/steps/check_answers_form.rb

```diff
@@ -23,11 +23,8 @@ def application_corrected
       end
 
       def application_changed_since_request?
-        last_updated_at > application.resubmission_requested
-      end
-
-      def last_updated_at
-        LastUpdateFinder.call(application)
```

Comment:
This is the only place where LastUpdateFinder is called, so if we're removing it here we should probably delete it entirely.